CHANGES.rst

```diff
@@ -8,6 +8,7 @@ Changelog
 - Remove ``broker.py`` and depreciated arg scheme related code.
 - Simplify the TChannel event types. Removed ``operational_error``,
   ``system_error`` and ``application_error``. Added ``on_error``
+- Remove ``CallableRequestHandler`` and ``BaseRequestHandler``.
```

Comment:
@abhinav @blampe @junchaowu  the changelog is for customers... git tells us what happens to our internals...